easybuild/framework/easyconfig/templates.py

```diff
@@ -94,9 +94,11 @@
      "as specify by the --sysroot configuration option"),
     ('mpi_cmd_prefix', "Prefix command for running MPI programs (with default number of ranks)"),
     ('cuda_compute_capabilities', "Comma-separated list of CUDA compute capabilities, as specified via "
-     "--cuda-compute-capabilities configuration option or via cuda_compute_capabilities easyconfig parameter"),
+     "--cuda-compute-capabilities configuration option or via cuda_cc easyconfig parameter"),
```

Comment:
@casparvl Maybe I'm missing something, but there's no `cuda_cc` easyconfig parameter in EasyBuild 4.x (nor in the `5.0.x` branch), so this change is incorrect?